pkg/cmd/server/api/v1/types.go

```diff
@@ -42,6 +44,10 @@ type NodeConfig struct {
 
 	// DockerConfig holds Docker related configuration options.
 	DockerConfig DockerConfig `json:"dockerConfig"`
+
+	// KubeletArguments are key value pairs that will be passed directly to the Kubelet that match the Kubelet's
+	// command line arguments.  These are not migrated or validated, so if you use them they may become invalid.
```

Comment:
Comment needs to mention that setting these values will override other NodeConfig options.